.gitignore

```diff
@@ -55,6 +55,9 @@ bower_components/
 /Cutup/slack_keys.php
 /getChannels.py
 
+# Lock Files
+package-lock.json
```

Comment:
You'll want this to be checked in so that everyone has the same versions when working on the site.